lib/stripe/api_operations/search.rb

```diff
@@ -12,6 +13,9 @@ def _search(search_url, filters = {}, opts = {})
           opts: opts
         )
       end
+
+      extend Gem::Deprecate
+      deprecate :_search, "request_stripe_object", 2024, 1
```

Comment:
IIRC, the second argument here is the alternative shown to the user that should be used (like in https://github.com/stripe/stripe-ruby/blob/a685a13a/lib/stripe/api_operations/save.rb#L73). Can you recreate the deprecation message locally to check what users will see?

Reply:
Added a test!